script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -227,122 +266,138 @@ Once the sun sets, things will start to get really hectic at the café.
 [sha:1c29987639387c283881c302fbff7687cc62e712]{
 -- Page 583, Offset 34941.
 -- わたしは少しだけ淋しい気持ちになって、去って行く彼の背中と広場の子供たちを眺めていた。
-Feeling a little lonely, I stare at his back and at the children playing in the square.
+Feeling a little lonely, I stare at his retreating back and at the children playing in the square.
 }
 [sha:cbadd8852c0394f83e6de0b2d811efb26ac565ca]{
 -- Page 584, Offset 34942.
 -- それがわたしと、わたしの町が生きていた頃の、最後の記憶。
+// Switching to past tense from here on, since there's a couple interjections
+// from the present that are kinda awkward if we promote all of this to present.
 That's my final memory from back when I still lived, and my hometown still existed.
```

Comment:
Not sure I see that second interpretation.

Re: tense, I had originally been promoting everything to present, but I ran into at least one line (can't immediately find it now) that reached in from the present, which then felt kinda weird when the rest of the scene was _also_ present. This juncture is the best place to change it imo since it's where you get the 'reveal', the descent into horror.    I could be convinced to go change it all back, but I would prefer a good way to keep the distinction between the 'present then' and 'present now'.

Reply:
After thinking it over again, yeah, I think this scene should probably be present tense, most of it. There are more places where past tense feels weird compared to present. I think some of them are like Noel coming in and commenting on it in this and one more place. Then there's the "I saw lots of things, I guess, but those are pretty easy to promote to present tense, or alternatively, could just be kept past tense as a sort of short-term flashback, even if that isn't super elegant.